test/FunctionsV2/WebJobs.Extensions.DurableTask.Tests.V2.csproj

```diff
@@ -9,13 +9,14 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.3.1" />
     <PackageReference Include="FluentAssertions" Version="4.19.4" />
+    <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.3.1" />
     <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.0" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Storage" Version="3.0.0" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Logging.ApplicationInsights" Version="3.0.0" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.0" />
     <PackageReference Include="Moq" Version="4.7.145" />
+    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
```

Comment:
Is this still needed? Can it be removed?

Reply:
removed.